_includes/pbc/all/install-features/202212.0/install-the-dynamic-multiple-stores.md

```diff
@@ -1126,12 +1126,16 @@ Make sure that the following changes have been applied in transfer objects:
  
 ### Change configuration 
 
-{% info_block warningBox "Configuration store.php" %}
+{% info_block warningBox "Configuration stores.php" %}
 
-Dynamic store allows not to use the configuration in the file `config/Shared/stores.php` where setup configureation for  store's. 
+Before dynamic store was introduced, the configuration for the store was stored in the file `config/Shared/stores.php`. Since the dynamic store is enabled, the configuration for the store is stored in the database, and the file `config/Shared/stores.php` deprecated. 
```

Comment:
```suggestion
Before dynamic store was introduced, configuration for the store was stored in the file `config/Shared/stores.php`. Since the dynamic store is now enabled, configuration for the store is stored in the database, making the file `config/Shared/stores.php` deprecated. 
```